sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_receiver_async.py

```diff
@@ -45,6 +45,7 @@
 from ._async_utils import create_authentication, get_running_loop
 
 if TYPE_CHECKING:
+    from .._common.auto_lock_renewer import AutoLockRenewer
```

Comment:
should this be the asnyc one?

Reply:
I'm surprised that didn't show up on mypy but maybe it only got hit in tests.  good catch.